app.js

```diff
@@ -1,8 +1,22 @@
 // import functions and grab DOM elements
 
+
+import { waterFalls } from './data.js';
+
+
+
 // let state
 
-// set event listeners 
-  // get user input
-  // use user input to update state 
-  // update DOM to reflect the new state
+
+const water = document.getElementById('water');
+for (let fall of waterFalls) {
+    const a = document.createElement('a');
```

Comment:
this code (lines 13-20) is doing the work to render all your homepage items - this needs to be abstracted into a function (that takes in an individual 'fall'), which you can call within your loop, and write a Test for